packages/devtools/kit/src/rsdoctor.ts

```diff
@@ -0,0 +1,66 @@
+import type { Manifest } from '@rsdoctor/types';
+
+export async function fetchShardingFile(url: string): Promise<string> {
+  const { Url } = await import('@rsdoctor/utils/common');
+  if (Url.isUrl(url)) {
+    const resp = await fetch(url);
+    return resp.text();
+  }
+  // json string
+  return url;
+}
+
+export async function parseManifest(
```

Comment:
This function may can migarate to rsdoctor repo, later. Would you want to do this ?